athlinks-schema.ttl

```diff
@@ -174,4 +193,107 @@ athlinks:chipTime
 athlinks:agePercentage
   a owl:DataTypeProperty ;
   rdfs:domain athlinks:IntervalResult;
-  rdfs:range schema:Decimal .
+  rdfs:range xsd:decimal .
+
+athlinks:isHidden
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:isVisible
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:isEnabled
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:isApproved
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:isHandicap
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:isNavigable
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:elevUnit
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:TerrainSpecifics;
+  rdfs:range xsd:float .
+
+athlinks:altitude
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:TerrainSpecifics;
+  rdfs:range xsd:float .
+
+athlinks:startMarked
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:finishMarked
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:distanceUnit
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:float .
```

Comment:
Lets try to keeps updates as minimal as possible. If we try to add every property they could ever want in one go, we will never get this thing merged.
So for now lets just add distance, and make the rdfs:range schema:QuantitiveValue which has a unit type and value.